frontend/src/components/workspaces/Templates/TemplatesTable/TemplatesTable.tsx

```diff
@@ -96,35 +63,34 @@ const TemplatesTable: FC<ITemplatesTableProps> = ({ ...props }) => {
       className="w-full flex-grow flex-wrap content-between py-0 overflow-auto scrollbar"
       style={{ height: 'calc(100vh - 380px)', minHeight: '171px' }} //465px if we want to add a Box footer
```

Comment:
I'd avoid hardcoding such kind of measures. If there's no way to calculate them from here it's better to put this as props I guess... (I'm mostly referring to the _380px_)

Reply:
actually i moved this style properties (height and min-height) to a CSS file. About the fixed value of `calc` i fear it must be a fixed value beacuse it's necessary to activate the scrollbar of the table. the final correct pixel value will be found when we insert all the components in a page with header (navbar) and footer. based on these last elements we can decide the correct value. Tell me if it could be the right way to proceed

Reply:
It could be better to use a css variable (that could be set inside a parent or something) then, or still make it settable somewhere... Otherwise can't we set it as full-height and contain it from outside?
The fact is if we have such kind of value that is decided inside the component but depends on something outside... It's not really a component...